features/facebookImport/src/components/advertisingValueMiniStory/advertisingValueMiniStory.jsx

```diff
@@ -14,9 +38,26 @@ const AdvertisingValueMiniStory = ({ randomAdInterests, numberInterests }) => {
                 }}
             />
             <ul>
-                {randomAdInterests.map((interest, index) => (
-                    <li key={index}>{interest}</li>
-                ))}
+                {randomAdInterests.map((interest, index) => {
+                    const refWidth = useRef(0);
+                    const fontSize = calculateFontSize(
```

Comment:
This does not really use the arguments, right? It could be taken outside the loop.

Reply:
Oh yes, forgot to remove it from there